### DISCUSSION TITLE
[Android] Expected behaviour when we change the interaction mode of the navigation bar (i.e. full screen gesture mode, 3-button navigation)

### DISCUSSION THREAD

**Discussion by NinjaDevRN** [react-navigation/react-navigation#11766] [Q&A]
I am struggle with this scenario:

1. Open an app
2. Set it to the background mode  (ex.: minimize the app)
3. Go to system settings
4. Change the navigation type of the navigation bar from "Swipe gestures" to "Buttons"
5. Open the app

<details><summary>Navigation type image</summary>
<img src="https://github.com/react-navigation/react-navigation/assets/126331230/c3d7edf5-806d-4097-91ec-60c2c8e8743d" alt="drawing" width="300"/></details>

1) Should the app restarts by default?
2) May be we need to handle that logic manually?
3) What is the best practice to handle that scenario?

**Comment by rakibuli2gether** [comment]
@NinjaDevRN i am also facing the same issue. any update on this ?
